assets/src/admin/site-scan-notice/plugins-with-amp-incompatibility.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * WordPress dependencies
+ */
+import { useContext } from '@wordpress/element';
+import { _n, sprintf } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { Plugins } from '../../components/plugins-context-provider';
+import { getPluginSlugFromFile } from '../../common/helpers/get-plugin-slug-from-file';
+
+/**
+ * Render a DETAILS element for each plugin causing AMP incompatibilities.
+ *
+ * @param {Object} props                               Component props.
+ * @param {Array}  props.pluginsWithAmpIncompatibility Array of plugins with AMP incompatibilities.
+ */
+export function PluginsWithAmpIncompatibility( { pluginsWithAmpIncompatibility } ) {
+	const { fetchingPlugins, plugins } = useContext( Plugins );
+
+	if ( fetchingPlugins ) {
+		return null;
+	}
+
+	const pluginNames = Object.values( plugins ).reduce( ( accumulatedPluginNames, plugin ) => {
+		return {
+			...accumulatedPluginNames,
+			[ getPluginSlugFromFile( plugin.plugin ) ]: plugin.name,
+		};
+	}, {} );
+
+	return (
+		<>
+			<p>
+				{ _n(
+					'AMP compatibility issues discovered with the following plugin:',
+					'AMP compatibility issues discovered with the following plugins:',
+					pluginsWithAmpIncompatibility.length,
+					'amp',
+				) }
+			</p>
+			{ pluginsWithAmpIncompatibility.map( ( pluginWithAmpIncompatibility ) => (
+				<details
+					key={ pluginWithAmpIncompatibility.slug }
+					className="amp-site-scan-notice__plugin-details"
+				>
+					<summary
+						className="amp-site-scan-notice__plugin-summary"
+						dangerouslySetInnerHTML={ {
+							__html: sprintf(
+								/* translators: 1: plugin name; 2: number of URLs with AMP validation issues. */
+								_n(
+									'<b>%1$s</b> on %2$d URL',
+									'<b>%1$s</b> on %2$d URLs',
+									pluginWithAmpIncompatibility.urls.length,
+									'amp',
+								),
+								pluginNames[ pluginWithAmpIncompatibility.slug ],
+								pluginWithAmpIncompatibility.urls.length,
+							),
+						} }
+					/>
+					<ul className="amp-site-scan-notice__urls-list">
+						{ pluginWithAmpIncompatibility.urls.map( ( url ) => (
+							<li key={ url }>
+								<a href={ url } target="_blank" rel="noopener noreferrer">
+									{ url }
+								</a>
```

Comment:
I guess it'd be best to use `ExternalLink` from `@wordpress/components` instead of a plain `<a>` tag.

Reply:
Scratch that. `ExternalLink` adds an icon next to a link and there seem to be no way to remove the icon at the JS-level (other than hiding it with CSS):

![Screenshot 2021-12-03 at 11 50 51](https://user-images.githubusercontent.com/478735/144591491-10794928-a3ae-4957-8cfb-5cac2be02b31.png)